specification/azuredependencymap/DependencyMap.Management/main.tsp

```diff
@@ -233,6 +246,33 @@ union SourceType {
   string,
 }
 
+@doc("Model representing the result of the export dependencies asynchronous operation.")
+model ExportDependenciesOperationResult {
+  @doc("The status URL of export dependencies operation.")
+  id?: string;
+
+  @doc("The resource name of the operation status. It must match the last segment of 'id' field.")
+  name?: string;
+
+  @doc("The overall arm status of the operation. It has one of the terminal states - Succeeded/Failed/Canceled.")
+  status: string;
+
+  @doc("Contains error details if status is Failed/Canceled.")
+  error?: Azure.ResourceManager.Foundations.ErrorDetail;
+
+  @doc("Properties for export dependencies. These should only be set if the status is Succeeded.")
+  properties?: ExportDependenciesResultProperties;
+}
+
+@doc("Model representing properties returned upon successful completion of the export dependencies asynchronous operation.")
+model ExportDependenciesResultProperties {
+  @doc("The SAS URI of the blob containing the exported dependencies data.")
+  exportedDataSasUri?: string;
```

Comment:
>exportedDataSasUri [](http://example.com/codeflow?start=2&length=18)

Add format: Uri. I think you can use a built-in Typespec type url. Ref: [https://typespec.io/docs/standard-library/built-in-data-types/#url](https://typespec.io/docs/standard-library/built-in-data-types/#url).